.tekton/rhtap-backstage-plugins-pull-request.yaml

```diff
@@ -217,6 +217,128 @@ spec:
             workspace: git-auth
           - name: netrc
             workspace: netrc
+      - name: generate-plugin-hash
+        params:
+          - name: ociStorage
+            value: $(params.output-image).script
+          - name: ociArtifactExpiresAfter
+            value: $(params.image-expires-after)
+          - name: SCRIPT_RUNNER_IMAGE
+            value: registry.redhat.io/rhel9/nodejs-20:latest
+          - name: SCRIPT
+            value: |
+              #!/usr/bin/env node
+              'use strict';
+              
+              const fs = require('fs');
+              const path = require('path');
+              
+              // Find all package.json files recursively
```

Comment:
Maybe reduce the scope to `/plugins` as we may add packages in the future as utility apps.